packages/cerebral-website-tutorial/parts/10_1_using_path/public/style.css

```diff
@@ -0,0 +1,17 @@
+.fadein {
+    -webkit-animation: fadein 0.5s;
+    animation: fadein 0.5s;
+}
+@-webkit-keyframes fadein {
+    from {opacity: 0;} 
+    to {opacity: 1;}
+}
+@keyframes fadein {
+    from {opacity: 0;}
+    to {opacity: 1;}
+}
+
```

Comment:
ok, thanks for the feedback
good idea with the load part1 script
having linting now in place not only linter is happy :)
working on the last chapters...